### PR TITLE
Summary page responsive UI

### PR DESCRIPTION
**Pull request by Andrii-Matselikh**
A third page with logic and responsive UI

### CODE REVIEW

File: src/app/components/homepage-components/navbar/input-field/input-field.component.html [hunk 1]
```diff
@@ -24,6 +24,6 @@
   <div class="add-item">
     <img src="../../assets/user.png" alt="user-logo">
     <img src="../../assets/shopping_basket.png" alt="basket">
-    <p class="cart-items">{{productDetailService.addToCartItems}}</p>
+    <p class="cart-items">{{this.productDetailService.addToCartItems}}</p>
```
Comment (SStegniy) [inline src/app/components/homepage-components/navbar/input-field/input-field.component.html:27]: do you need <this> before productDetailService ??

File: src/app/components/order-page-components/order-page/order-page.component.ts [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+import { BreadcrumbsService } from '../../../services/breadcrumbs.service';
+import { countriesAndCities } from './countries-cities-constant';
+
+@Component({
+  selector: 'app-order-page',
+  templateUrl: './order-page.component.html',
+  styleUrls: ['./order-page.component.scss']
+})
+
+@HostListener('window:resize', ['$event'])
```
Comment (SStegniy) [inline src/app/components/order-page-components/order-page/order-page.component.ts:14]: Did you use this event somewhere?

File: src/styles.css [hunk 3]
```diff
@@ -5,7 +5,7 @@ html {
   margin: 0 auto;
   max-width: 1260px;
   font-family: Open Sans, Poppins;
-  border: 1px solid dodgerblue;
+  /*border: 1px solid dodgerblue;*/
```
Comment (SStegniy) [inline src/styles.css:8]: remove comment code

**Review by SStegniy — APPROVED** [review]

File: src/app/components/homepage-components/navbar/input-field/input-field.component.html (hunk 1 above)
Comment (Andrew1995bono) [reply src/app/components/homepage-components/navbar/input-field/input-field.component.html:27]: Fixed using *ngIf directive. Now, when there are no added products, the basket is empty, so the number is not reflected.

File: src/app/components/order-page-components/order-page/order-page.component.ts (hunk 2 above)
Comment (Andrew1995bono) [reply src/app/components/order-page-components/order-page/order-page.component.ts:14]: HostListener decorator has been removed due to no usage.

File: src/styles.css (hunk 3 above)
Comment (Andrew1995bono) [reply src/styles.css:8]: +Removed unused CSS code.

**Pull request merged**